pkg/config/config.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/kelseyhightower/envconfig"
+)
+
+// ChaosServerConfig defines the configuration for Chaos Dashboard
+type ChaosServerConfig struct {
+	ListenHost           string `envconfig:"LISTEN_HOST" default:"0.0.0.0"`
+	ListenPort           int    `envconfig:"LISTEN_PORT" default:"2333"`
+	MetricAddress        string `envconfig:"METRIC_ADDRESS"`
+	EnableLeaderElection bool   `envconfig:"ENABLE_LEADER_ELECTION"`
+	Database             *DatabaseConfig
```

Comment:
I want to start some discussion about our configuration. Do we really want to use configurations as env vars? This is not common currently, I would say better to use a config file, or just flags. Any thoughts? @cwen0  @fewdan  @Yisaer

Reply:
In my opinion, if we are running these components in a Kubernetes environment, using environment variables is not much different from using a configuration file. In addition, if we think that using config configuration files is better in the future, we can replace them with configuration files. This change is tiny.

Reply:
That's might not be a tiny change. It will cause some back-compatibility issues, so it is hard to change one to another way.
Env vars and flags are not much different. But using the config file approach is quite useful if we want to have something like hot reload in the future (is it possible?).

Reply:
But I think it is okay for now, I don't want to add more work to you. So we can accept this first or change this in another PR if it is really needed.

Reply:
I think there is no essential difference between the two. You can maintain the status quo first and make changes if necessary.